packages/dynamic-flows/src/step/cameraStep/cameraCapture/screens/CameraNotSupported/CameraNotSupported.messages.ts

```diff
@@ -9,8 +9,8 @@ export default defineMessages({
   paragraph: {
     id: 'dynamicFlows.CameraCapture.CameraNotSupported.paragraph',
     defaultMessage:
-      'The browser you are using does not have support for a camera. Please try a different browser, device, or download our app',
+      "The browser you’re using doesn't have support for a camera. Try a different browser, device, or download our mobile app.",
```

Comment:
```suggestion
      "The browser you're using doesn't have support for a camera. Try a different browser, device, or download our mobile app.",
```

I noticed we’re not using the same style of apostrophe 👍